demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,70 @@
+<h5>Navigate through options in the popup</h5>
+
+<table class="table mt-4">
+  <tbody>
+    <tr>
+      <td><kbd>ArrowUp</kbd></td>
+      <td>Highlight above option</td>
+    </tr>
+    <tr>
+      <td><kbd>ArrowDown</kbd></td>
+      <td>Highlight below option</td>
+    </tr>
+    <tr>
+      <td><code>mouseenter</code></td>
+      <td>Highlight option under the mouse pointer</td>
+    </tr>
+  </tbody>
+</table>
+
+<p>
+  Note that starting keyboard navigation will always start from the currently activated option, no matter if activated from the keyboard or the mouse.
```

Comment:
Stating the obvious, also this is not the typeahead user documentation.

Reply:
For the "obvious" part, see my main comment above. Regarding the user documentation aspect, the whole "interactions" section IS end user documentation. So what do we do?